### PR TITLE
added links to aid understanding

### PR DESCRIPTION
**Pull request by sshanks-kx**

### CODE REVIEW

File: docs/kb/callbacks.md [hunk 1]
```diff
@@ -48,7 +48,7 @@ Next, ensure that the “remote” function on the server is expecting the name
 q)proc:{[arg1;  ;argn ; callname]  }
 ```
 
-Finally, in the remote function obtain the open handle of the calling process from the system variable `.z.w`. Use this link back to the caller to invoke the callback function.
+Finally, in the remote function obtain the open handle of the calling process from the system variable [`.z.w`](../ref/dotz.md#zw-handle). Use this link back to the caller to invoke the callback function.
```
Comment (JosieMD) [inline docs/kb/callbacks.md]:
```suggestion
Finally, in the remote function, obtain the open handle of the calling process from the system variable [`.z.w`](../ref/dotz.md#zw-handle). Use this link back to the caller to invoke the callback function.
```

File: docs/kb/multithreaded-input.md [hunk 2]
```diff
@@ -12,9 +12,9 @@ By default, kdb+ is single-threaded, and processes incoming queries sequentially
...
 
-There can be a maximum of 1020 concurrent connections, with each connection requiring a minimum of 64MB, the real amount depending on the working space required by the query being executed. Each connection has its own thread, which is either reading, calculating or writing a response; in addition there is the main thread, which monitors stdin, invokes `.z.ts` on timer expiry and monitors other socket descriptors. (There should not be any). 
+There can be a maximum of 1020 concurrent connections, with each connection requiring a minimum of 64MB, the real amount depending on the working space required by the query being executed. Each connection has its own thread, which is either reading, calculating or writing a response; in addition there is the main thread, which monitors stdin, invokes [`.z.ts`](../ref/dotz.md#zts-timer) on timer expiry and monitors other socket descriptors. (There should not be any).
```
Comment (JosieMD) [inline docs/kb/multithreaded-input.md]:
```suggestion
There can be a maximum of 1020 concurrent connections, with each connection requiring a minimum of 64MB, the real amount depending on the working space required by the query being executed. Each connection has its own thread, which is reading, calculating or writing a response. In addition, there is the main thread, which monitors stdin, invokes [`.z.ts`](../ref/dotz.md#zts-timer) on timer expiry and monitors other socket descriptors. (There should not be any). 
```

File: docs/kb/multithreaded-input.md [hunk 3]
```diff
@@ -12,9 +12,9 @@ By default, kdb+ is single-threaded, and processes incoming queries sequentially
...
 
-Updates to globals are allowed only if they occur from within `.z.ts`, or via a socket listed in `.z.W`, and even then they should not be frequent, as these wait for completion of exiting queries, blocking new queries (using multiple-read single-write lock). If an attempt is made to update globals from threads other than main, they should get a `'no update` error.
+Updates to globals are allowed only if they occur from within [`.z.ts`](../ref/dotz.md#zts-timer), or via a socket listed in [`.z.W`](../ref/dotz.md#zw-handles), and even then they should not be frequent, as these wait for completion of exiting queries, blocking new queries (using multiple-read single-write lock). If an attempt is made to update globals from threads other than main, they should get a `'no update` error.
```
Comment (JosieMD) [inline docs/kb/multithreaded-input.md]:
```suggestion
Updates to globals are allowed only if they occur from within [`.z.ts`](../ref/dotz.md#zts-timer), or via a socket listed in [`.z.W`](../ref/dotz.md#zw-handles). Updates should not be frequent, as they wait for completion of exiting queries and block new queries (using multiple-read single-write lock), thus slowing processing speeds. If an attempt is made to update globals from threads other than main, a `'no update` error is issued.
```

**Review by JosieMD — COMMENTED** [review]
Please check thoroughly. I may have changed the meaning of this text.

**Comment by sshanks-kx** [comment]
changes made, thanks

**Pull request merged**